packages/connectivity/example/test_driver/connectivity.dart

```diff
@@ -22,6 +22,8 @@ void main() {
         case ConnectivityResult.wifi:
           expect((_connectivity.getWifiName()), completes);
           expect((await _connectivity.getWifiIP()), isNotNull);
+          // Returns null on emulators so we just check that it doesn't throw
+          await _connectivity.getWifiBSSID();
```

Comment:
```suggestion
          expect((_connectivity. getWifiBSSID()), completes);
```

Reply:
good catch, you're right, this is more consistent with the rest of the test